filament/src/driver/opengl/OpenGLDriver.cpp

```diff
@@ -2488,6 +2488,7 @@ void OpenGLDriver::beginFrame(uint64_t monotonic_clock_ns, uint32_t frameId) {
             }
         }
     }
+    mContextManager.setPresentationTime(monotonic_clock_ns);
```

Comment:
I don't know if this clock is correct. What's passed here is the steady_clock and I don't think this one has anything to do with the system time. What clock does this API expect?

Wouldn't it be easier to NOT add a new method and instead just do it where we do eglSwapBuffers() and use the system time?